packages/core/src/validation.ts

```diff
@@ -189,7 +189,9 @@ const customNotificationUpdate = Joi.object({
   eventCode: Joi.string(),
   message: Joi.string().required(),
   id: Joi.string().required(),
-  autoDismiss: Joi.number()
+  autoDismiss: Joi.number(),
+  onclick: Joi.function(),
```

Comment:
@Adamj1232 maybe should have caught this sooner but can we make it camel case?? `onClick`

Reply:
@taylorjdawson the only reason i didn't was that Notify v1 didn't use camel case and didn't want to break backwards compatibility. We can update but what are your thoughts there?